packages/flutter/lib/src/scheduler/scheduler.dart

```diff
@@ -3,16 +3,35 @@
 // found in the LICENSE file.
 
 import 'dart:async';
-import 'dart:ui' show VoidCallback;
+import 'dart:developer';
```

Comment:
It's needed for the `Timeline` calls. (I just copied it from the animation scheduler).